ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -1173,10 +1144,120 @@ impl<'a> Assemble<'a> {
         }
     }
 
+    fn cg_traceloopjump(&mut self) {
+        // Loop the JITted code if the `tloop_start` label is present.
```

Comment:
Why wouldn't it be present?

Reply:
This comment was already there, I just copied it. But an example for a trace without a `tloop_start` label are side-traces.